protocol/testutil/constants/affiliates.go

```diff
@@ -0,0 +1,32 @@
+package constants
+
+import (
+	affiliates "github.com/dydxprotocol/v4-chain/protocol/x/affiliates/types"
+)
+
+var (
+	DefaultAffiliateTiers = affiliates.AffiliateTiers{
```

Comment:
We can put this outside of `testutil` and just regular constants. You can then use this object in the upgrade handler too.